marimbabot_vision/scripts/attention_viz.py

```diff
@@ -0,0 +1,85 @@
+"""
+This script is used to visualize the attention maps of the model,
+showing which parts of the image are used to generate each token.
+"""
+
+import matplotlib
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import matplotlib.cm as cm
+
+import numpy as np
+import torch
+
+from PIL import Image
+from transformers import (DonutProcessor, VisionEncoderDecoderConfig,
+                          VisionEncoderDecoderModel)
+
+INPUT_FILE = './demo.png'
+MODEL = './model'
```

Comment:
Maybe use the huggingface model path/repo, so, that we anyone can use this model for detection right away.